server-spi/build.gradle

```diff
@@ -0,0 +1,17 @@
+plugins {
+    id "com.flipkart.varadhi.java-published-library-conventions"
+    id 'java-test-fixtures'
+}
+
+dependencies {
+    api(project(':entities'))
+    api(project(':spi'))
+
+    api(libs.vertx.core)
```

Comment:
does this work?

Reply:
this is not a module name. this is a syntax wrt using version catalog. Refer https://docs.gradle.org/current/userguide/version_catalogs.html for details. 

server-spi is the module (containing interfaces) which has dependency on vertx, or whichever framework we would be using. Framework independent interfaces would be defined in spi. 

I will add package-info.java file.